File: math/GCD/Python/GCD.py
def gcd(a,b):
    if(b==0): 
        return a 
    else: 
        return gcd(b,a%b)
    
a = int(input("Enter the first integer: "))
b = int(input("Enter the second integer: "))
    
print("The greatest common denominator of " + str(a) + " and " + str(b) + " is: " + str(gcd(a,b)))

File: math/GCD/Matlab/gcd.m
val1 = input('Enter first value: ');
val2 = input('Enter second value: ');

if val1 > val2
    temp = val1;
    val1 = val2;
    val2 = temp;
end

fprintf('\nGreatest common denominator of %d and %d: %d', val1, val2, gcd(val1, val2));

function a = gcd(val1, val2)
    if val2 == 0
        a = val1;
    else
        a = gcd(val2, mod(val2, val1));
    end
end